assets/templates/signup.html

```diff
@@ -41,46 +41,46 @@ <h5>Regístrate</h5>
       <form class="signup-form">
         <fieldset>
           <legend>Datos personales</legend>
-          <ul class="holder-group">
-            <li>
-              <ul class="holder-group">
-                <li class="holder-item holder-item--small">
+          <ul class="field-group">
+            <li class="field-item">
+              <ul class="field-group">
+                <li class="field-item field-item--50 m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
                   <label for="name">Nombre:</label>
                   <input type="name" name="name" id="name" placeholder="Nombre">
-                <li class="holder-item holder-item--small">
+                <li class="field-group field-item--50 m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
                   <label for="lastname">Apellido:</label>
                   <input type="lastname" name="name" id="lastname" placeholder="Apellido">
                 </li>
               </ul>
             </li>
-            <li class="holder-item holder-item--big">
+            <li class="field-item m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
               <label for="phone">Teléfono:</label>
               <input type="tel" name="phone" id="phone" placeholder="9 11 2345-6789">
             </li>
           </ul>
         </fieldset>
         <fieldset>
-          <ul class="holder-group">
+          <ul class="field-group">
             <legend>Usuario y contraseña</legend>
-            <li class="holder-item holder-item--big">
+            <li class="field-item m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
               <label for="email">Email:</label>
               <input type="email" name="email" id="email" placeholder="Tu Email será tu nombre de usuario">
             </li>
-            <li class="holder-item holder-item--big">
+            <li class="field-item m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
               <label for="password">Contraseña:</label>
               <input type="password" name="password" id="password" placeholder="Escribe tu clave secreta">
             </li>
           </ul>
         </fieldset>
         <fieldset>
           <legend>Politica de privacidad y promociones</legend>
-          <ul class="holder-group">
-            <li class="holder-item holder-item--big checkbox">
+          <ul class="field-group">
+            <li class="field-item field-item__checkbox m-bottom--lv5"><!-- margin-bottom: 30px m-bottom--lv5 -->
```

Comment:
What's the difference between this `field-item__checkbox` and this `m-bottom--lv5` in terms of use of BEM, maybe we should stick to only one way of use. What do you think?

Reply:
The difference they have is that "--lv-5" is a modifier and "__checkbox" is an element the field-item, the BEM convention says that it is block__element--modifier, with the name the block or element or modifier separate with a "-"